cli/src/test.rs

```diff
@@ -358,17 +358,15 @@ fn run_tests(
 
             if attributes.skip {
                 println!(
-                    "{:>3}.  {}",
-                    opts.test_num,
+                    "     ⌀ {}",
```

Comment:
Why remove the format specifier?

Reply:
`test_num` isn't incremented for skipped tests, so currently if several tests are skipped in a row, it looks like
```
      1.  skipped test 1
      1.  skipped test 2
      1.  skipped test 3
      1.  skipped test 4
      1.  skipped test 5
      1.  skipped test 6
      1. ✓ test that runs 1
      2. ✓ test that runs 2
```

I think it looks better to hide the test number in this case.

If you'd prefer to have the test number incremented for all tests, including skipped ones, I can make that change.

Reply:
ah that's a good catch - let's have the test number incremented always, because we can parse tests with `tree-sitter parse -n <test_number>`, so knowing the number always is helpful

Reply:
Commenting to follow because the changes to test reports have made my json report PR no longer viable and I'll have to update its behavior once this is merged